packit/cli/packit_base.py

```diff
@@ -41,7 +41,19 @@
 logger = logging.getLogger("packit")
 
 
-@click.group("packit", context_settings=get_context_settings())
+class Aliased(click.Group):
```

Comment:
nice, sounds like a much better solution to me than copy-pasta the function

Reply:
Great! will do the pre-commit check.